### ISSUE TITLE
[Bug] Chat Channel Feature Incompatible with DiscordSRV

### ISSUE THREAD

**Issue by ghost** [nulli0n/SunLight-spigot#43]
# Description
Messages send on "Global" Channel are forwarded to DiscordSRV's Global Channel but the same is not true for other channels.

# Steps to Reproduce
1. Configure channels on both DiscordSRV and SunLight
2. Send a message on Discord in the configured channel [Alternatively send a message in Minecraft in the configured channel]
3. The message is not forwarded to Minecraft/Discord respectively
 
# Server Info
```
Current: git-Purpur-2095 (MC: 1.20.2)*
DiscordSRV version 1.27.0
SunLight version 3.10.1
NexEngine version 2.2.12
```

# Configs
![image](https://github.com/nulli0n/SunLight-spigot/assets/97077949/77b3923b-a064-4ed6-a01b-7fad9516053c)
https://paste.gg/p/anonymous/de95fd3961204778b9f5a88347c08eb0

**Comment by UFend** [comment]
Actual problem

**Comment by nulli0n** [comment]
Fixed in v3.12.3
